nncf/torch/quantization/algo.py

```diff
@@ -1057,12 +1075,23 @@ def is_weights(ip: PTTargetPoint) -> bool:
                                         'Please take a look at the documentation for a detailed information.'
                                         .format(quantizers_with_overflow_fix_str))
 
+        matching_qspec_info = self._qspec_comp_state_parameters.get(str(primary_qid), None)
```

Comment:
None is returned by default for `get`

Reply:
State is not intended for direct access like this by design.
State should be passed to `load_state` or `from_state` methods only that know how to parse this state. You're breaking encapsulation here.